docs/using-workflows/adding-triggers.md

```diff
@@ -0,0 +1,92 @@
+# Adding Triggers to Workflows
+
+## Overview
+
+Triggers in Relay cause workflows to be run. They come in several flavors. In order of increasing sophistication, they are:
+
+* **Schedule triggers** - Relay includes a `cron` style service which can trigger workflows on a time-based schedule.
+* **Push triggers** - External systems can send events to a Relay API endpoint with a workflow-specific authentication token, causing their associated workflow to run.
+* **Webhook triggers** - Relay supports external services that implement webhook callbacks. It generates a unique URL to which the external service POSTs a payload; Relay then runs a user-defined container to handle the request.
+
+The [Workflow reference guide](../reference/relay-workflows.md#triggers) defines the syntax for triggers. This guide shows how to use each kind in conjunction with an external system and how to set them up on your Relay account.
+
+All varieties of triggers are defined in a top-level `triggers:` section of your workflow YAML. The value of `triggers:` is a list of maps, each of which is a separate trigger that will cause the workflow to run. A single workflow can have multiple triggers to accomodate different event sources.
+
+Every trigger definition needs a `name` field and a `source` map that specifies the `type` of trigger it is.  Currently `schedule`, `webhook`, and `push` are the only valid values for `type`. Which additional fields are needed depends on the `type`.
+
+Trigger definitions also need a [`binding`](../reference/relay-workflows.md#binding), a map whose keys are the names of parameters the workflow requires. The values can come from the event payload, extracted using the [`!Data` custom type](../reference/relay-types.md#data). This allows you to dynamically inject from the trigger into the workflow.
+
+## Schedule triggers
+
+Relay has an internal scheduling service that will kick off workflow runs on time-based schedules - think of it as a fancy cloud-based `crontab`. A schedule trigger has the simplest definition, because it just adds a cron format `schedule` key to the `source` map that defines the trigger:
+
+```yaml
+triggers:
+- name: my-schedule-trigger
+  source:
+    type: schedule
+    schedule: '7-57/10 * * * *'
```

Comment:
May be worth noting that the reason you need to quote this is because putting a `*` at the front will really confuse the YAML parser if you don't.